Assignment2/EnduEo-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,44 @@
+{	
+	"@context":{
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"subjects": "http://www.w3.org/2001/UPM/subjects/",
+		"teachers": "http://www.w3.org/2001/UPM/teachers/",
+		"name": "rdfs:label",
+		"lastName": "rdfs:label",
+		"hometown": "rdfs:label",
+		"age": "rdfs:label",
+		"alma mater": "rdfs:label",
+		"momName": "rdfs:label",
+		"dadName": "rdfs:label",
+		"description": "rdfs:comment"
+	},
+	"name":"Endurance Eghosa",

```

Comment:
Add the "@id" and the "@type" definitions. It is important to provide an identifier and specify the type of resource. Thus, machines will be able to know that this resource refers to a Person, in this case you :-)

Reply:
Thanks, I've updated the files. 
One question, could I specify the @id of the resource as a string? For example my name
"@id": "Endurance"  or should it always be an URI?

Reply:
It must be always a URI.